File: 2A_2/cpp/funkcje01.cpp
/*
 * funkcje01.cpp
 */


#include <iostream>
#include <cmath>
using namespace std;

void sumuj(int a, int b) {
	int suma = a + b;
    cout << "Suma: " << suma << endl;
}


int potega(int podstawa, int wykladnik) {
    int wynik = pow(podstawa, wykladnik);
    return wynik;
}

int main(int argc, char **argv)
{
	int liczba1 = 0;
	int liczba2 = 0;
	cout << "Podaj liczby: " << endl;
	cin >> liczba1;
	cin >> liczba2;

	sumuj(liczba1, liczba2);
	cout << potega(liczba1, liczba2) << endl;

	return 0;
}

File: 2A_2/python/funkcje01.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  funkcje01.py
#  

def sumuj(a, b):  # definicja funkcji z parametrami
    suma = a + b
    print("Suma:", suma)


def odejmij(a, b):
    roznica = a - b
    print("Różnica:", roznica)


def potega(podstawa, wykladnik):
    wynik = podstawa**wykladnik  # użycie operatora potęgowania
    return wynik


def main(args):
    liczba1 = int(input("Podaj liczbę: "))
    liczba2 = int(input("Podaj liczbę: "))
    # sumuj(liczba1, liczba2)  # wywołanie funkcji z argumentami
    # odejmij(liczba1, liczba2)
    wynik = potega(liczba1, liczba2)
    print(wynik)
    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
